src/jwt/__mocks__/keycloak-js.ts

```diff
@@ -32,7 +32,7 @@ class Keycloak {
     this.responseMode = 'fragment';
     this.responseType = 'code';
     this.flow = 'standard';
-    this.clientId = ITLessKeycloak() ? 'console-dot' : 'cloud-services';
+    this.clientId = ITLess() ? 'console-dot' : 'cloud-services';
```

Comment:
Not blocking - but we have this exact same check/string here: https://github.com/RedHatInsights/insights-chrome/pull/2708/files#diff-345249b2500e3d310d8235c95d24191bb7a2dc823e0a6f4d2a8e4da2b45a697aR68

Might be worth creating a clientId helper so we don't have to change it in multiple locations in the future.

Reply:
Yeah, I agree, will do that when Martin's changes are merged